src/frontend/next/src/components/Post/Post.tsx

```diff
@@ -0,0 +1,210 @@
+import React, { useRef, useState } from 'react';
+
+import useSWR from 'swr';
+import 'highlight.js/styles/github.css';
+import { makeStyles } from '@material-ui/core/styles';
+import { Box, Grid, Typography, ListSubheader, createStyles } from '@material-ui/core';
+
+// For style, need to add:
+// import '../styles/telescope-post-content.css';
+
+interface Props {
```

Comment:
The next.js docs/examples on using TypeScript suggest using `type` vs. `interface` for Props.